src/gui/DatabaseWidget.cpp

```diff
@@ -686,9 +686,12 @@ void DatabaseWidget::openUrlForEntry(Entry* entry)
             }
         }
     } else {
-        QString urlString = entry->webUrl();
+        QString urlString = entry->url();
         if (!urlString.isEmpty()) {
-            QDesktopServices::openUrl(urlString);
+            QString scheme = urlString.split(':')[0];
+            QUrl url = QUrl(urlString);
+            if (!scheme.isEmpty()) url.setScheme(scheme);
```

Comment:
wrap the code block with { }
Shouldn't QUrl already set the scheme based on a url of that structure?